src/plugins/ui_actions/public/plugin.ts

```diff
@@ -33,6 +38,7 @@ export class UiActionsPlugin implements Plugin<UiActionsSetup, UiActionsStart> {
     setTheme(core.theme);
     this.service.registerTrigger(rowClickTrigger);
     this.service.registerTrigger(visualizeFieldTrigger);
+    this.service.registerTrigger(visualizeEditorTrigger);
```

Comment:
is there a reason this is not registered from visualize editor plugin ?

Reply:
@ppisljar I added this here to be consistent with the `this.service.registerTrigger(visualizeFieldTrigger);` that navigates from Discover to Lens

Reply:
I moved it to the visualizations plugin :)

Reply:
I moved it to the visualizations plugin :)